lib/integrations/hip_chat.rb

```diff
@@ -1,83 +1,68 @@
-module Integrations
-  class HipChat < Base
-    # NOTE: HipChat integration development still underway
 
-    def message_text
-      message = self.send(event_type.dup.concat("_message").to_sym)
-      run_description = run[:description].present? ? ": #{run[:description]}" : ""
-      "Your Rainforest Run (<a href=\"#{payload[:frontend_url]}\">Run ##{run[:id]}#{run_description}</a>) #{message}"
-    end
-
-    def run_completion_message
-      "is complete!"
-    end
-
-    def run_error_message
-      "has encountered an error!"
-    end
+class Integrations::HipChat < Integrations::Base
+  def self.key
+    "hip_chat"
```

Comment:
Oh, we automatically camelize? That's kind of annoying, it's always referred to as `hipchat` in lower case (e.g. in the gem and whatnot). But if we do it automatically I guess it's OK.